lib/iris/fileformats/um_cf_map.py

```diff
@@ -16,11 +16,11 @@
 # along with Iris.  If not, see <http://www.gnu.org/licenses/>.
 #
 # DO NOT EDIT: AUTO-GENERATED
-# Created on 12 January 2017 09:01 from 
+# Created on 03 May 2018 12:05 from 
 # http://www.metarelate.net/metOcean
-# at commit c8b9eaf6733843702297f9770e166bc9ad756235
+# at commit 21a706f9d3c123007a2cdf4b681b0e42eb8e79da
 
-# https://github.com/metarelate/metOcean/commit/c8b9eaf6733843702297f9770e166bc9ad756235
+# https://github.com/metarelate/metOcean/commit/21a706f9d3c123007a2cdf4b681b0e42eb8e79da
```

Comment:
I couldn't find this commit. Any ideas?

Reply:
hello @pelson 

thanks for picking this up, 

I have pushed this commit to github, it should have been done prior to the PR going up, but it didn't trigger

let me know if there's anything else you need

marqh